tools/executionPoker/bin.js

```diff
@@ -8,38 +8,14 @@ const ganache = require('ganache-cli');
 
 const GAS_LIMIT = 0xfffffffffffff;
 
-let submissionCounter = 0;
+// let submissionCounter = 0;
 
 class MyExecutionPoker extends ExecutionPoker {
   onSlashed (execId) {
     this.log(`got slashed, executionId(${execId})`);
     // we are done
     process.exit(0);
   }
-
-  async submitProof (disputeId, computationPath) {
-    await super.submitProof(disputeId, computationPath);
-
-    submissionCounter++;
-
-    if (submissionCounter === 2) {
```

Comment:
Also, if we account my first comment, we have to keep this or be smarter to check this 😁.